backend/src/kitconcept/intranet/upgrades/v20250828001/configure.zcml

```diff
@@ -0,0 +1,15 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    >
+  <genericsetup:upgradeSteps
+      profile="kitconcept.intranet:default"
+      source="20250825001"
+      destination="20250828001"
+      >
+    <genericsetup:upgradeDepends
+        title="Adds kitconcept.intranet.cc_email, default_email and allowed domains to the control panel registry."
+        import_steps="controlpanel,plone.app.registry"
```

Comment:
```suggestion
        import_steps="plone.app.registry"
```

We don't need the controlpanel import step because you didn't touch controlpanel.xml -- that's only needed when adding or removing an entire control panel.

Reply:
done